lib/robots/dor_repo/gis_assembly/extract_boundingbox.rb

```diff
@@ -131,87 +136,105 @@ def dd2ddmmss_abs(orig_val)
           "#{degrees}#{QDEG}" + (minutes > 0 ? "#{minutes}#{QMIN}" : '') + (seconds > 0 ? "#{seconds}#{QSEC}" : '')
         end
```

Comment:
It doesn't look like dd2ddmmss_abs is being used anymore? Is it the case that this coordinate conversion was only needed in MODS and not Cocina, so it was half removed?